### ISSUE TITLE
Tubular shadows

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9813]
**Preferred term label:**
Tubular shadows

**Synonyms**
Tubular ghosts

**Definition (free text, please give PubMed ID)**
This is the most severe form of NOA with total absence of cells inside the seminiferous tubules. Instead of cells, hyaline, an amorphous basement membrane-like material is found.

**Parent term (use hpo.jax.org/app)**
HP:0011961

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 399771

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
is this separate from  Sertoli cells only syndrome ? Maybe we could add this to the definition of the latter?

**Comment by MagoWyr** [comment]
Yes, this is a different phenotype than Sertoli cell only phenotype. Please don't use the term Sertoli cell only syndrome, as this is not a syndromic condition.

Definition: Tubular shadows are characterised by a complete absence of cells in the testicular tubules, meaning that germ cells as well as somatic Sertoli cells are absent. The cells have been replaced by hyalinization.
________________________________

**Comment by pnrobinson** [comment]
@MagoWyr 
added as
Tubular shadows
HP:0034945
